scripts/data/regenerate_tests.sh

```diff
@@ -63,4 +68,9 @@ done
 for test_case in "${full_test_cases[@]}"; do
     echo "Generating test data: full mode, chain state @ $test_case, single block"
     generate_test "full_fast" $test_case
```

Comment:
`full_fast` or `full`?

Reply:
`full`, fast mode requires downloading data which does not make much sense for tests (at least for now)